requirements2.txt

```diff
@@ -0,0 +1,64 @@
+argon2-cffi==20.1.0
+async-generator==1.10
+attrs==20.3.0
+backcall==0.2.0
+bleach==3.3.0
+cffi==1.14.5
+colorama==0.4.4
+cycler==0.10.0
+decorator==4.4.2
+defusedxml==0.7.1
+entrypoints==0.3
+ipykernel==5.5.0
+ipython==7.21.0
+ipython-genutils==0.2.0
+jedi==0.18.0
+Jinja2==2.11.3
+jsonschema==3.2.0
+jupyter-client==6.1.12
+jupyter-contrib-core==0.3.3
+jupyter-contrib-nbextensions==0.5.1
+jupyter-core==4.7.1
+jupyter-highlight-selected-word==0.2.0
+jupyter-latex-envs==1.4.6
+jupyter-nbextensions-configurator==0.4.1
+jupyterlab-pygments==0.1.2
+kiwisolver==1.3.1
+lxml==4.6.2
+MarkupSafe==1.1.1
+matplotlib==3.3.4
+mistune==0.8.4
+nbclient==0.5.3
+nbconvert==6.0.7
+nbformat==5.1.2
+nest-asyncio==1.5.1
+notebook==6.2.0
+numpy==1.20.1
+packaging==20.9
+pandas==1.2.3
+pandocfilters==1.4.3
+parso==0.8.1
+pickleshare==0.7.5
+Pillow==8.1.2
+prometheus-client==0.9.0
+prompt-toolkit==3.0.17
+pycparser==2.20
+Pygments==2.8.1
+pyparsing==2.4.7
+pyrsistent==0.17.3
+python-dateutil==2.8.1
+pytz==2021.1
+pywin32==300
+pywinpty==0.5.7
+PyYAML==5.4.1
+pyzmq==22.0.3
+scipy==1.6.1
+seaborn==0.11.1
+Send2Trash==1.5.0
+six==1.15.0
+terminado==0.9.2
+testpath==0.4.4
+tornado==6.1
+traitlets==5.0.5
+wcwidth==0.2.5
+webencodings==0.5.1
```

Comment:
requirements2.txt worked fine! either in this pr or next, we may as well replace the old requirements.txt with this one.

Reply:
> we may as well replace the old requirements.txt with this one.

Didn't want to touch it yet bc I thought you had done some GIS work.

Reply:
Ah, ok. I did do some GIS work with geopandas, but it was mostly experimental / scratch code. Since it's not included in any of our commits, should be fine to replace the old requirements.txt for now. We can add geopandas back when we get to that point